charts/fluent-operator/values.yaml

```diff
@@ -245,6 +245,54 @@ fluentbit:
 #        port: 2020
 #        addLabels:
 #          app: "fluentbit"
+    
+#    # Loki output. Uncomment to produce a ClusterOutput, to be encapsulated in fluentbit config
+#    # See https://github.com/fluent/fluent-operator/blob/master/docs/plugins/fluentbit/output/loki.md
+#    loki:
```

Comment:
```
{{ if .Values.Kubernetes -}}
{{ if .Values.fluentbit.enable -}}
{{ if .Values.fluentbit.output.loki -}}
```
Because of this, shouldn't we set `enable=true`?default (setting) enable=false`?

Reply:
like this https://github.com/fluent/fluent-operator/blob/master/charts/fluent-operator/values.yaml#L208

Reply:
Hi @wenchajun thanks for your review

My understanding is that the loki.enable=true key is redundant/not needed.

Why? Because the 

' {{ if .values.fluentbit.loki}} 

Does the same thing. If there is a node in the values.yaml for that loki key and it contains something then the loki manifests will be produced. If not they wont.

If a user wanted to keep their loki node in their values.yaml but disable the manifest generation - why not just remove the loki node instead? Better to remove what is unused rather than leaving it as clutter. If they want an old version of history - that would be a job for git. The enable key supports the user to keep unused yaml content (bad) - i couldnt discern any other purpose. So i removed it.

Hope that helps

Reply:
Hi @wenchajun 

I have explored this a little further - it looks like the defacto convention is to have a .values.fluentbit.output.<output> node in the values.yaml file with defaults... Thus requiring the .values.fluentbit.output.loki.enabled key with a default value of false. 

In this case I have that node commented out in values.yaml - the user has to uncomment it and set their own values (implicitly enabling it).

Happy to add the .values.fluentbit.output.loki.enabled key (behaviour of which i assume as switching on/off the associated manifest generation) if yourself or other reviewers would like to enforce convention of having an enabled key and .values.fluentbit.output.<output> node uncommented (as opposed to no enabled key and it commented out - as it is now). Let me know

Reply:
I first just thought the formatting should be uniform, but you have a point that the `values.yaml` should be more concise, and I agree with you.

Reply:
> Happy to add the .values.fluentbit.output.loki.enabled key (behaviour of which i assume as switching on/off the associated manifest generation) if yourself or other reviewers would like to enforce convention of having an enabled key and .values.fluentbit.output. node uncommented (as opposed to no enabled key and it commented out - as it is now). Let me know

@WaywardWizard Thanks for this PR. Comment or uncomment a large section of code is not as easy as turning a switch on. I prefer to add `.values.fluentbit.output.loki.enabled` and set it to `false` by default

Reply:
> Happy to add the .values.fluentbit.output.loki.enabled key (behaviour of which i assume as switching on/off the associated manifest generation) if yourself or other reviewers would like to enforce convention of having an enabled key and .values.fluentbit.output. node uncommented (as opposed to no enabled key and it commented out - as it is now). Let me know

@WaywardWizard Thanks for this PR. Comment or uncomment a large section of code is not as easy as turning a switch on or off. I prefer to add `.values.fluentbit.output.loki.enabled` and set it to `false` by default

Reply:
Okay understood - thanks for reviewing. Ill make the necessary changes